content/developer/howtos/upgrade_custom_db.rst

```diff
@@ -0,0 +1,281 @@
+Upgrading a customized database
+===============================
+
+
+.. todo:: toctree
+..     :titlesonly:
+..     :glob:
+
+..     upgrade/*
+
+
+Upgrading to a new version of Odoo can be challenging, especially if the database you work on
+contains custom modules. This page intent is to explain the technical process of upgrading a
+database with customized modules. For a standard overview, please refer to the
+:doc:`Upgrade documentation </administration/upgrade>`.
+
+We consider a custom module, any module that extends the standard code of Odoo and that was not
+built with the Studio app. 
+Before upgrading those modules, or before asking for them to be upgraded, have a look at the
+:ref:`upgrade/sla` to make sure who's responsible for it.
+
+While working on what we refer to as the **custom upgrade** of your database, keep in mind the goals
+of an upgrade:
+
+#. Stay supported
+#. Get the latest features
+#. Enjoy the performance improvement
+#. Reduce the technical debt
+#. Benefit from security improvements
+
+With every new version of Odoo, changes are introduced. These changes can have an impact modules on
+which customization have been developed. This is the reason why upgrading a database that contains
+custom modules requires additional steps in order to upgrade their source code.
+
+These are the steps to follow to upgrade customized databases:
+
+#. :ref:`Stop the devolopments and challenge them <upgrade_custom/stop_developments>`.
+#. :ref:`upgrade_custom/request_upgrade`.
+#. :ref:`Make your module installable on an empty database <upgrade_custom/empty_database>`.
+#. :ref:`Make your module installable on the upgraded database <upgrade_custom/upgraded_database>`.
+#. :ref:`Test extensively and do a rehearsal <upgrade_custom/testing_rehearsal>`.
+#. :ref:`Upgrade in production <upgrade_custom/production>`.
+
+
+.. _upgrade_custom/stop_developments:
+
+Stop the developments
+=====================
+
+Starting an upgrade requires commitment and development resources need to be allocated to it. If
+developments keep being made at the same time, those features will need to be re-upgraded and tested
+everytime you change them. This is why we recommend a complete freeze of the codebase when starting
+the upgrade process. Needless to mention that bug fixing is exempt from this recommendation.
+
+Once you have stopped development, it is a good practice to assess the developments made and compare
+them with the features introduced between you current version and the version you are targeting.
+Challenge the developments as much as possible, find functionnal workarounds.
+Removing redundancy between your developments and the standard version of Odoo will lead to an eased
+upgrade process and reduce the technical debt.
+
+.. note::
+   You can find information on the changes between versions in the `release notes
+   <https:/odoo.com/page/release-notes>`_. Those may seem light at first but they will help point
```

Comment:
The sentence `Those may seem light at first but ...` stood out when reading this, IMO it clashes with the style of the other sentences and the global tone of the page. I don't think we should introduce any biais towards the release notes and let the reader decide if they feel like they are light or not IMO